target/linux/ramips/image/mt76x8.mk

```diff
@@ -19,6 +19,16 @@ endef
 DEVICE_VARS += TPLINK_FLASHLAYOUT TPLINK_HWID TPLINK_HWREV TPLINK_HWREVADD TPLINK_HVERSION
 
 
+define Device/axing_w06
+  DTS := W06
+  IMAGE_SIZE := 15040k
+  DEVICE_TITLE := Axing W06
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-ledtrig-netdev
```

Comment:
Drop the kmod-ledtrig-netdev. Without an usb led, this trigger doesn't make much sense.